### PR TITLE
843577 - Update HTML to PDF troubleshooting details - Hotfix

### PR DESCRIPTION
**Pull request by SowmiyaLoganathan**
Hi All, 

I have updated the HTML to PDF troubleshooting details. Kindly review this and merge. 

Regards,
Sowmiya L

### CODE REVIEW

File: File-Formats/PDF/Convert-HTML-To-PDF/troubleshooting.md [hunk 1]
```diff
@@ -424,6 +424,56 @@ blinkConverterSettings.CommandLineArguments.Add("--disable-setuid-sandbox");
 </tr>
 </table>
 
+## Mac Machine Error: "Exception has been thrown by the target of an invocation"
```
Comment (PrakashViswanath) [inline File-Formats/PDF/Convert-HTML-To-PDF/troubleshooting.md:427]: OCR not working on Mac: Exception has been thrown by the target of an invocation

**Review by PrakashViswanath — CHANGES_REQUESTED** [review]
Kindly change the title and proceed.

File: File-Formats/PDF/Convert-HTML-To-PDF/troubleshooting.md (hunk 1 above)
Comment (SowmiyaLoganathan) [reply File-Formats/PDF/Convert-HTML-To-PDF/troubleshooting.md]: Updated the title

**Comment by SowmiyaLoganathan** [comment]
rebuild

**Comment by SowmiyaLoganathan** [comment]
rebuild

**Review by PrakashViswanath — APPROVED** [review]
The changes are fine.

**Pull request merged**